packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -466,7 +466,23 @@ export class DataStores implements IDisposable {
 			return;
 		}
 
-		assert(!!context, 0x162 /* "There should be a store context for the op" */);
+		if (context === undefined) {
+			// Former assert 0x162
+			throw DataProcessingError.create(
+				"No context for op",
+				"processFluidDataStoreOp",
+				message,
+				{
+					local,
+					messageDetails: JSON.stringify({
+						type: message.type,
+						contentType: typeof message.contents,
```

Comment:
Are we allowed to log contents of a message?

Reply:
No. But we're not logging the contents, only the `typeof`

Reply:
See https://github.com/microsoft/FluidFramework/blob/ec1307589e69016ba4f6f3a1d34763e2fa4826a7/packages/utils/telemetry-utils/src/error.ts#L209 for instance where we only log some properties.

Reply:
I don't understand your feedback. We already log extractSafePropertiesFromMessage automatically from the DataProcessingError logic.

Reply:
I missed the `typeof` there! I thought we are logging the contents here, my bad.